source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BaseAdDummyClient.cs

```diff
@@ -24,7 +24,8 @@ namespace GoogleMobileAds.Unity
     public class BaseAdDummyClient
     {
         protected static DummyAdBehaviour AdBehaviour = new GameObject().AddComponent<DummyAdBehaviour>();
-        protected static GameObject prefabAd, dummyAd = null;
+
+        protected GameObject prefabAd, dummyAd = null;
```

Comment:
Why is this no longer static?

Reply:
Didn't allow for multiple ads showing at once